src/objective-c/GRPCClient/private/NSDictionary+GRPC.m

```diff
@@ -33,7 +33,8 @@
 
 #import "NSDictionary+GRPC.h"
 
-#include <grpc.h>
+#include <grpc/grpc.h>
```

Comment:
nit: No need to include it in the implementation if we include it in the header.